geo/src/main/java/org/odk/collect/geo/selection/SelectionMapFragment.kt

```diff
@@ -176,7 +177,7 @@ class SelectionMapFragment(
         }
 
         if (showNewItemButton) {
-            binding.newItem.setOnClickListener {
+            binding.newItem.setMultiClickSafeOnClickListener {
```

Comment:
I would also fix `binding.layerMenu` above. There it's probably also possible to open multiple dialogs?